src/math_functions.cpp

```diff
@@ -674,6 +674,35 @@ double maxwell_spectrum(double T) {
 }
 
 
+double gaussian_spectrum(double mean, double standard_deviation) {
+  double r2 = 1.01;
+  double v1 = 0.;
+  double v2 = 0.;
```

Comment:
No need to declare these outside of the `while` loop. Down below, I'd just have `double v1 = ...`